Source/Diagnostics/BoostedFrameDiagnostic.cpp

```diff
@@ -917,13 +1025,15 @@ writeMetaData ()
     BL_PROFILE("BoostedFrameDiagnostic::writeMetaData");
 
     if (ParallelDescriptor::IOProcessor()) {
-
-        if (!UtilCreateDirectory(WarpX::lab_data_directory, 0755))
-            CreateDirectoryFailed(WarpX::lab_data_directory);
+        //const std::string fullpath = WarpX::lab_data_directory + "/snapshots";
+        const std::string fullpath = WarpX::lab_data_directory ;
+        if (!UtilCreateDirectory(fullpath, 0755))
+            CreateDirectoryFailed(fullpath);
 
         VisMF::IO_Buffer io_buffer(VisMF::IO_Buffer_Size);
         std::ofstream HeaderFile;
         HeaderFile.rdbuf()->pubsetbuf(io_buffer.dataPtr(), io_buffer.size());
+        //std::string HeaderFileName(WarpX::lab_data_directory + "/snapshots/Header");
```

Comment:
Could we get rid of this line?

Reply:
The new location for the back-transformed lab-frame diagnostic of the full domain is in lab_fame_data/snapshots, so the header file location is written in the same directory. 
 std::string HeaderFileName(WarpX::lab_data_directory + "/snapshots/Header");
I have removed the previous path ( std::string HeaderFileName(WarpX::lab_data_directory + "/Header");)